docker-compose.yaml

```
@@ -34,19 +35,41 @@ services:
       - 5432:5432
     environment:
       - POSTGRES_USER=user
       - POSTGRES_PASSWORD=pass
-      - POSTGRES_DB=evolution
+      - POSTGRES_DB=evolution_db
       - POSTGRES_HOST_AUTH_METHOD=trust
     volumes:
-      - postgres_data:/var/lib/postgresql/data
-    expose:
-      - 5432
+    - postgres_data:/var/lib/postgresql/data
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)